### PR TITLE
🚧 Provide proposal of KEDA governance

### PR DESCRIPTION
**Pull request by tomkerkhove**
Provide proposal of governance for KEDA project.

Adding 🚧 until all maintainers have aye'd.

Depends on #662 
Relates to #660

### CODE REVIEW

**Comment by caniszczyk** [comment]
This LGTM. You also may want to add a section on adding/removing projects.

For example, say you want to add something like https://github.com/kedacore/dashboard or a kedacore/utilities project or transfer a repo into kedacore from somewhere else, that should be voted upon by the existing maintainers, similar to adding a new maintainer.

**Comment by tomkerkhove** [comment]
@yaron2 PTAL, I've added project management governance proposal as per @caniszczyk his request

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+[super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of
+the current maintainers.
+
+## Managing Projects
+
+New projects can be added to the organization after a successful
+[super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote)
+vote of the existing maintainers.
+
+Contributors who want to donate a scaler, sample or other contribution must
```
Comment (yaron2) [inline GOVERNANCE.md]: Makes sense for scalers, I'm just wondering if the vote process for contributions in general is scalable.

Comment (tomkerkhove) [reply GOVERNANCE.md]: It's more talking about contributing new repos in our org, what about this:
```suggestion
Contributors who want to donate an add-on scaler, sample or donate a repo must
```

**Comment by tomkerkhove** [comment]
PTAL @jeffhollan  @bbrowning @ahmelsayed @anirudhgarg  @caniszczyk

**Comment by caniszczyk** [comment]
LGTM

**Comment by caniszczyk** [comment]
'git commit --amend -s' and push -f it back and we can close this back

**Comment by tomkerkhove** [comment]
_That moment when you forget to sign off your commit and you mess up the whole PR_

**Comment by tomkerkhove** [comment]
Clean PR is available on https://github.com/kedacore/keda/pull/690

**Pull request closed without merging**